### ISSUE TITLE
Question about training iterations for other configurations

### ISSUE THREAD

**Issue by yaramohamadi** [thuml/DomainGuidance#1]
Hi Connor,

Thank you for sharing the code for your super interesting work! 
I am trying to reproduce your results in the appendix of your paper like for the DiT-L-2-300M model, or for the same DiT-XL-2-700M model but for the 512x512 image size. I have a question about the setup you have used for these results. Are you using the same setup and also stopping at the 24000 iteration? 

Thanks a lot :) 
Yara

**Comment by ConnorZhong** [comment]
For all experiments involving fine-tuned models, unless otherwise specified, we follow the same setup, using the same learning rate and stopping at 24,000 iterations.

**Comment by yaramohamadi** [comment]
Thank you for your help!
